components/earnings/commons/earnings_base.lua

```diff
@@ -106,14 +106,16 @@ function Earnings.calculate(conditions, year, mode, perYear, divisionFactor)
 
 	local lpdbQueryData = mw.ext.LiquipediaDB.lpdb('placement', {
 		conditions = conditions,
-		query = 'sum::prizemoney, mode',
+		query = 'mode, ' .. (divisionFactor == nil and 'sum::individualprizemoney' or 'sum::prizemoney'),
```

Comment:
Extract to a function

Reply:
done, if you have a better function name for it let me know^^